### PR TITLE
fix(docs) strings

### PR DESCRIPTION
**Pull request by talbii**

### CODE REVIEW

File: docs/command-reference/strings/bitpos.md [hunk 1]
```diff
@@ -54,21 +52,19 @@ However, this behavior changes if you are looking for clear bits and specify a r
 dragonfly> SET mykey "\xff\xf0\x00"
 "OK"
 dragonfly> BITPOS mykey 0
-(integer) 0
+(integer) 12
```
Comment (ashotland) [inline docs/command-reference/strings/bitpos.md:55]: this seems wrong

Comment (romange) [reply docs/command-reference/strings/bitpos.md:55]: @talbii  - was it fixed?

Comment (talbii) [reply docs/command-reference/strings/bitpos.md:55]: It looks correct to me, the string "\xff\xf0\x00" in binary is `1111'1111'1111'0000..`, and the first position in which we have `0` is `12`. I also checked with Redis and it seems to agree.

**Review by romange — COMMENTED** [review]
setrange diff is screwed. what happened there? 
https://github.com/dragonflydb/documentation/pull/32/files?short_path=b3d5005#diff-fc48d0b321a3d5bb9ec2a41999a5e983b4793c329851b7875af83928a59936b1

File: docs/command-reference/strings/bitpos.md (hunk 1 above)
Comment (ashotland) [reply docs/command-reference/strings/bitpos.md:55]: sorry - :)

**Comment by talbii** [comment]
> setrange diff is screwed. what happened there? https://github.com/dragonflydb/documentation/pull/32/files?short_path=b3d5005#diff-fc48d0b321a3d5bb9ec2a41999a5e983b4793c329851b7875af83928a59936b1

I've inserted a <NUL> character (i.e., ASCII zero) to demonstrate zero padding, I guess the GH preview doesn't like that.

The site renders fine on my end, should I replace the character with something like "<NUL>" or leave it as-is? (ps., it renders as &#0;, a question mark on my end).

**Review by romange — APPROVED** [review]

**Pull request merged**